docker-compose.yaml

```
@@ -0,0 +1,14 @@
+services:
+  db:
+    image: 'mariadb:10.10.2'
```

This speeds the db up:
```suggestion
    image: 'mariadb:10.10.2'
    tmpfs:
      - /tmp
      - /var/lib/mysql
```